Update configure.ac and Makefile.am to resolve warnings/errors

Signed-off-by: Mark Hatle <mark.hatle@windriver.com>

diff -ur tnftp-20100108.orig/configure.ac tnftp-20100108/configure.ac
--- tnftp-20100108.orig/configure.ac	2010-01-12 00:57:31.000000000 -0600
+++ tnftp-20100108/configure.ac	2013-06-04 07:28:22.124548804 -0500
@@ -62,7 +62,9 @@
 #
 # Checks for programs.
 #
+AM_PROG_AR
 AC_PROG_CC
+AM_PROG_CC_C_O
 AC_PROG_AWK
 AC_PROG_LIBTOOL
 
diff -ur tnftp-20100108.orig/libnetbsd/Makefile.am tnftp-20100108/libnetbsd/Makefile.am
--- tnftp-20100108.orig/libnetbsd/Makefile.am	2010-01-04 00:24:20.000000000 -0600
+++ tnftp-20100108/libnetbsd/Makefile.am	2013-06-04 07:29:28.272548832 -0500
@@ -5,7 +5,7 @@
 libnetbsd_la_SOURCES =
 
 
-CPPFLAGS = \
+AM_CPPFLAGS = \
 	-I$(srcdir) \
 	-I$(top_srcdir) \
 	-I$(top_builddir)
